internal/kubernetes/clients/issuers.go

```diff
@@ -224,3 +233,79 @@ func NewAWSPCAClusterIssuerClient(config *rest.Config) (*Generic[*awspca.AWSPCAC
 
 	return genericClient, nil
 }
+
+// NewKMSIssuerClient returns an instance of a generic client for querying
+// KMS Issuers
+func NewKMSIssuerClient(config *rest.Config) (*Generic[*kmsissuerapi.KMSIssuer, *kmsissuerapi.KMSIssuerList], error) {
+	genericClient, err := NewGenericClient[*kmsissuerapi.KMSIssuer, *kmsissuerapi.KMSIssuerList](
+		&GenericClientOptions{
+			RestConfig: config,
+			APIPath:    "/apis",
+			Group:      kmsissuerapi.GroupVersion.Group,
+			Version:    kmsissuerapi.GroupVersion.Version,
+			Kind:       "kmsissuers",
+		},
+	)
+	if err != nil {
+		return nil, fmt.Errorf("error creating generic client: %w", err)
+	}
+
+	return genericClient, nil
+}
+
+// NewVenafiEnhancedIssuerClient returns an instance of a generic client for querying
+// Venafi enhanced issuers
+func NewVenafiEnhancedIssuerClient(config *rest.Config) (*Generic[*veiapi.VenafiIssuer, *veiapi.VenafiIssuerList], error) {
+	genericClient, err := NewGenericClient[*veiapi.VenafiIssuer, *veiapi.VenafiIssuerList](
+		&GenericClientOptions{
+			RestConfig: config,
+			APIPath:    "/apis",
+			Group:      veiapi.SchemeGroupVersion.Group,
+			Version:    veiapi.SchemeGroupVersion.Group,
+			Kind:       "venafiissuers",
+		},
+	)
+	if err != nil {
+		return nil, fmt.Errorf("error creating generic client: %w", err)
+	}
+
+	return genericClient, nil
+}
+
+// NewVenafiEnhancedClusterIssuerClient returns an instance of a generic client for querying
+// Venafi enhanced cluster issuers
+func NewVenafiEnhancedClusterIssuerClient(config *rest.Config) (*Generic[*veiapi.VenafiClusterIssuer, *veiapi.VenafiClusterIssuerList], error) {
+	genericClient, err := NewGenericClient[*veiapi.VenafiClusterIssuer, *veiapi.VenafiClusterIssuerList](
+		&GenericClientOptions{
+			RestConfig: config,
+			APIPath:    "/apis",
+			Group:      veiapi.SchemeGroupVersion.Group,
+			Version:    veiapi.SchemeGroupVersion.Group,
```

Comment:
Here should be `veiapi.SchemeGroupVersion.Version`

Reply:
Thanks, good spot, addressed in a3bec0050df2f0eacb355dbab4145c71e0a8b8bf